### ISSUE TITLE
Change current site to Kids section

### ISSUE THREAD

**Issue by ghost** [EcoStarsClub/letsgo#10]
Convert the current version of the site into a kids only section. Since it already kind of is kid-friendly with the apps, teh games, ethe books, etc, I say we convert it into the kids only section, and add a Grown Ups link in the menu.  The grown ups section will be more about our organization, our mission, programs, etc 

@shellyperlman do yo agree?
LIke this:
![screen shot 2015-09-20 at 10 00 25 pm](https://cloud.githubusercontent.com/assets/14277703/9984410/d73249a4-5fe4-11e5-9a6b-14d791a8ec48.png)

**Comment by shellyperlman** [comment]
on the long run, yes i agree. right now we have nothing to put there for kids and we should not concentrate to create anything for it at this moment since we have the other projects to get off the ground: School program with videos. The Eco Stars APP - the Hub for Eco and Animal friendliness. - doing these will actually give us material to put content up there and then we can make it for kids I believe. -    let  me know what you were thinking, if you had another idea, etc. or if you had an idea of what to put up there now.

**Comment by ghost** [comment]
I agree we shouldn't create more products right now without the proper funding, but we do have some products we can put up so a) we get them out to more people and b) build users and c) show prospective sponsors we are out in community doing something (in addition to all the good stuff you've been doing with the events! :))

**Comment by shellyperlman** [comment]
When we were at the science center we got a lot of interest about the APP having all these places where you can be active or learn how to be eco and animal friendly - we can start collecting these organizations on our site - refer to and from them to our site.  - doing the Eco Stars Club is possible - I can send a letter to our users to see who can go to the science center for our first eco stars club.

**Comment by ghost** [comment]
Nice! so we can do this for now:
For games: the app
For books: the water ebook
For tv: the promo video
Club: the science center

**Comment by shellyperlman** [comment]
for club: Join the club or create a club in your area - we will need to just make up a program of 12 meetings and what can be done in each

Also: I would like to start listing on the site organizations so that not only we can later apprach them to send people to us adn we can send to them. but also for eco stars to be able to go by state and see what organizations exist that they can go learn or participate 

we can start collecting them there by state
